docs/manage/manage-subscription/cloud-flex-accounts.md

```diff
@@ -134,17 +131,13 @@ To view the Account page, do the following:
 
 1. Log in to your Sumo Logic Cloud Flex account.
 1. In the left navigation bar of the UI, select **Administration** > **Account**. The Account Overview tab of the Account page is shown by default.
-
-    :::note
-    You must have a role that grants you the Account Overview capability to view the Account Overview tab.
-    :::
-
+:::note
+You must have a role that grants you the Account Overview capability to view the Account Overview tab.
```

Comment:
I'd maybe link to the /manage capabilities doc where it explains them

Reply:
Done.